src/main/webapp/app/course/competencies/forms/course-competency-form.component.ts

```
@@ -122,14 +116,14 @@
         }
         let initialTitle: string | undefined = undefined;
-        if (this.isEditMode && this.formData && this.formData.title) {
+        if (this.isEditMode() && this.formData && this.formData.title) {
             initialTitle = this.formData.title;
         }
         this.form = this.fb.nonNullable.group({
             title: [
                 undefined as string | undefined,
                 [Validators.required, Validators.maxLength(255)],
-                [titleUniqueValidator(this.courseCompetencyService, this.courseId, initialTitle)],
+                [titleUniqueValidator(this.courseCompetencyService, this.courseId()!, initialTitle)],
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)